### PR TITLE
Added the "Holiday Definition File" for Colombia

### PR DESCRIPTION
**Pull request by glendel**
Added the "Holiday Definition File" for Colombia.

### CODE REVIEW

File: co.yaml [hunk 1]
```diff
@@ -0,0 +1,512 @@
...
+# - https://en.wikipedia.org/wiki/Public_holidays_in_Colombia
+# - https://publicholidays.co/
+# - https://www.calendariodecolombia.com/
+# - https://www.officeholidays.com/countries/colombia/
+# - https://www.timeanddate.com/holidays/colombia/
+# - http://www.bank-holidays.com/country/Colombia_42.htm
+---
+methods:
+  # Movable holiday: when they do not fall on a Monday, these holidays are observed the following Monday.
+  to_following_monday_if_not_monday:
```
Comment (ttwo32) [inline co.yaml]: Where is this method called?
I cannot find it.

File: co.yaml [hunk 2]
```diff
@@ -0,0 +1,512 @@
...
+    regions: [co]
+    function: independence_of_cartagena( year )
+  12:
+  - name: Día de la Inmaculada Concepción
+    regions: [co]
+    mday: 8
+  - name: Navidad
+    regions: [co]
+    mday: 25
+months:
```
Comment (ttwo32) [inline co.yaml]: Why are months duplicated?
Please check L94-L163.

File: co.yaml [hunk 3]
```diff
@@ -0,0 +1,512 @@
...
+    regions: [co]
+    function: independence_of_cartagena( year )
+  12:
+  - name: Día de la Inmaculada Concepción
+    regions: [co]
+    mday: 8
+  - name: Navidad
+    regions: [co]
+    mday: 25
+tests:
```
Comment (ttwo32) [inline co.yaml]: Multi year tests are good. 👍

File: co.yaml [hunk 4]
```diff
@@ -0,0 +1,512 @@
+# Colombian holiday definitions for the Ruby Holiday gem.
+#
+# Updated: 2018-06-01.
+#
+# Sources:
+# - https://en.wikipedia.org/wiki/Public_holidays_in_Colombia
```
Comment (ttwo32) [inline co.yaml]: Thank you for teaching sites. 😃 
Following users (including me) can know about Colombia holiday.

Comment (glendel) [reply co.yaml]: Sure, my pleasure! :blush:

**Comment by ttwo32** [comment]
@glendel OK. I would merge this PR. Thanks!!

**Pull request merged**